### DISCUSSION TITLE
Charts.css.py, a Python chart framework built on top of charts.css!

### DISCUSSION THREAD

**Discussion by rayluo** [ChartsCSS/charts.css#54] [Show and tell]
<sub>Since [the welcome message](https://github.com/ChartsCSS/charts.css/discussions/16) encourages us to "share ideas" and "engage with other community members", and there is even this "Show and tell" category, I am writing to share my project `charts.css.py`.<sub>

`Charts.css.py` is a Python chart framework built on top of charts.css!

* Similarity: The chart engine of `charts.css.py` *is* based on `charts.css`, so that output chart's look-and-feel is identical to `charts.css`.
* Difference: However, unlike `charts.css`'s *consuming* html table and providing a CSS api, `charts.css.py` focuses on *generating* the html table from your Python datasets, and providing a Python api.

You can know more from [`charts.css.py`'s online document and samples](https://rayluo.github.io/charts.css.py/).

**Comment by ramiy** [comment]
Added to https://github.com/ChartsCSS/awesome
